web/src/main/webapp/table.ts

```diff
@@ -626,45 +628,53 @@ export class TableView extends RemoteTableObjectView
         }
     }
 
+    private hLogLog(colName: string): void {
+	let rr = this.createHLogLogRequest(colName);
+        rr.invoke(new HLogLogReceiver(this.getPage(), rr, "HLogLog",
+				      (res) => alert("Distinct values in column \'" +
```

Comment:
Let's not use alerts.
Try first with reportError, and if that does not work well we'll find something else.